### PR TITLE
DEV-307 Working with environment variables

### PR DESCRIPTION
**Pull request by smoliver**
Secrets moved from `web-app/app.config.js` to environment variables.  Environment variables can either be set as system variables, as needed for netlify, or in a root `.env` file for development purposes.  

C4Coin `.env` file can be found in the Development Confluence space

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,5 @@
+GA_TRACKING_ID=
```
Comment (davesag) [inline .env.example:1]: leave these out if you are not going to set values

File: .gitignore [hunk 2]
```diff
@@ -41,7 +41,5 @@ jspm_packages/
 # built artifacts
 build
 
-# App Config Secrets
-web-app/app.config.js
-
 .DS_Store
+dist/
```
Comment (davesag) [inline .gitignore:45]: remember to add `.env` to `.gitignore` — you'll only use it for development.  On Netlify you'll set the environment variables via their web interface

File: web-app/app.config.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  mailchimp: {
+    u: process.env.MAILCHIMP_USER,
+    id: process.env.MAILCHIMP_ID,
+    url: 'https://c4coin.us12.list-manage.com/subscribe/post'
+  },
+  campaignMonitor: {
+    id: process.env.CAMPAIGN_MONITOR_ID,
+    communityForm: {
+      email: 'cm-skjjhl-skjjhl',
```
Comment (davesag) [inline web-app/app.config.js:25]: don't put these in here.  this is a public repo remember.

File: web-app/app.config.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      userId: 'cm-f-jtnlyy'
```
Comment (davesag) [inline web-app/app.config.js:26]: as above

File: .env.example [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+SQUIDEX_CLIENT_SECRET=
```
Comment (davesag) [inline .env.example:2]: add these to the readme with explanations of what they are and where to get them, and any defaults.

**Review by davesag — CHANGES_REQUESTED** [review]

**Comment by smoliver** [comment]
@davesag Are you available to talk, I think that there may be some confusion.  The .env is ignored, the .env.example servers to throw a warning if one of those variables isn't set in .env (or as system variables)

email & user id aren't secrets, just id's used by campaign-monitor

**Comment by davesag** [comment]
@smoliver Hey sam, sorry yes my confusion.  I misread the file name.  will approve now.
all good

**Review by davesag — APPROVED** [review]

**Pull request merged**